### ISSUE TITLE
Images are missing captions

### ISSUE THREAD

**Issue by TheAgencyatUF** [The-Agency-at-UF/on-the-pulse#38]
Currently, images in articles are missing captions and `aria-label` screen reader captions. A possible solution could be to manually add captions to each image and use these captions directly as the `aria-label` for each photo.
